DeviceInfo/DeviceInfoJsonRpc.cpp

```diff
@@ -33,13 +33,15 @@ namespace Plugin {
         Property<SysteminfoData>(_T("systeminfo"), &DeviceInfo::get_systeminfo, nullptr, this);
         Property<Core::JSON::ArrayType<AddressesData>>(_T("addresses"), &DeviceInfo::get_addresses, nullptr, this);
         Property<SocketinfoData>(_T("socketinfo"), &DeviceInfo::get_socketinfo, nullptr, this);
+        Property<StbcapabilitiesData>(_T("stbcapabilities"), &DeviceInfo::get_stbcapabilities, nullptr, this);
```

Comment:
Lets just call this capabilities, avoid alliteration :-) The fact that it is on the plugin Device, already explains that these are "device" -> STB capabilities and not Display capabilities since these can be found on the DisplayInfo :-)